pkg/cluster/majorversionupgrade.go

```diff
@@ -106,6 +106,34 @@ func (c *Cluster) removeFailuresAnnotation() error {
 	return nil
 }
 
+func (c *Cluster) criticalOperationLabel(pods []v1.Pod, remove bool) error {
```

Comment:
when I only see the call I would expect `c.criticalOperationLabel(pods, true)` to add the label and `c.criticalOperationLabel(pods, false)` to remove it. One can think of also two wrapper functions like `addCriticalOperationLabel` and `removeCriticalOperationLabel` which call this func here.

Reply:
wanted some code deduplication 🥲